lib/Dialect/HW/Transforms/FlattenIO.cpp

```diff
@@ -93,11 +93,22 @@ struct InstanceOpConversion : public OpConversionPattern<hw::InstanceOp> {
       }
     }
 
-    // Create the new instance...
-    Operation *targetModule = SymbolTable::lookupNearestSymbolFrom(
-        op, op.getReferencedModuleNameAttr());
+    // Get the new module return type.
+    llvm::SmallVector<Type> newResultTypes;
+    for (auto oldResultType : op.getResultTypes())
+      if (auto structType = getStructType(oldResultType))
```

Comment:
nit: if nesting is so deep, can we at least wrap the outer for in braces?

Reply:
Wouldn't call this a nit - the style guide is pretty explicit in requiring this to be braced https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements.